uint16 prediction_horizon   # [milliseconds]
uint16 sampling_period      # [milliseconds]
uint8 num_predictions
Prediction[] predictions